tests/trustless_gateway_car_test.go

```diff
@@ -631,6 +631,13 @@ func TestTrustlessCarEntityBytes(t *testing.T) {
 
 func TestTrustlessCarOrderAndDuplicates(t *testing.T) {
 	dirWithDuplicateFiles := car.MustOpenUnixfsCar("trustless_gateway_car/dir-with-duplicate-files.car")
+	multiblockCIDs := []string{
+		"bafkreie5noke3mb7hqxukzcy73nl23k6lxszxi5w3dtmuwz62wnvkpsscm",
+		"bafkreih4ephajybraj6wnxsbwjwa77fukurtpl7oj7t7pfq545duhot7cq",
+		"bafkreigu7buvm3cfunb35766dn7tmqyh2um62zcio63en2btvxuybgcpue",
+		"bafkreicll3huefkc3qnrzeony7zcfo7cr3nbx64hnxrqzsixpceg332fhe",
+		"bafkreifst3pqztuvj57lycamoi7z34b4emf7gawxs74nwrc2c7jncmpaqm",
+	}
```

Comment:
- Could you provide a recipe for the car file (in a README.md, there should be other examples)
- There is no way to avoid hardcoding these?

Reply:
@laurentsenta re: hardcoding https://github.com/ipfs/gateway-conformance/pull/87#discussion_r1253723076